### PR TITLE
docs: modify how-to/spec documentations

### LINKED ISSUE #99: docs: provide more information in `How-To/cli/README.md`
## Summary

Provide more information about `BatchAuction` and `PlaceBid` section in `How-To/cli/README.md`. Currently, content is lacking.

### PR DESCRIPTION
**Pull request by jeonghoyeo7**
## Description

closes: #99 

## Tasks

- [x] Modify README.md for CLI in how-to
- [x] Modify other documents in how-to 
- [x] Update spec documents
- [X] Modify some documentations from changing FixedPriceAuction as #130 

## References

- 

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Appropriate labels applied
- [ ] Targeted PR against correct branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://go.dev/blog/godoc).
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: docs/How-To/cli/README.md [hunk 1]
```diff
@@ -86,79 +93,38 @@ fundraisingd tx fundraising create-fixed-price-auction auction.json \
...
-      "granter": ""
-    }
-  },
-  "signatures": [
-    "UahsRZ27hATh0xu7M/IWFvNvaFESpQ+W0RmQhQql3ERsnYdTDrFP81/MxyYxuX4WNBUv4+3FyhOwEQ7hqlU+MQ=="
-  ]
-}
+### CreateBatchAuction
+
+An auctioneer can create a batch auction by setting the following parameters. Differently from a fixed price auction,  start_price does not affect the determination of the matched price, but is provided by the auctioneer as a reference price to bidders. See the [spec](https://github.com/tendermint/fundraising/blob/main/x/fundraising/spec/01_concepts.md#auction-types) for a detailed and technical information about a batch auction type.
```
Comment (jaybxyz) [inline docs/How-To/cli/README.md]:
```suggestion
An auctioneer can create a batch auction by setting the following parameters. Differently from a fixed price auction, `start_price` does not affect the determination of the matched price, but is provided by the auctioneer as a reference price to bidders. See the [spec](https://github.com/tendermint/fundraising/blob/main/x/fundraising/spec/01_concepts.md#auction-types) for a detailed and technical information about a batch auction type.
```

File: docs/How-To/cli/README.md [hunk 2]
```diff
@@ -59,19 +78,7 @@ In this JSON example, an auctioneer plans to create a fixed price auction that p
...
-| start_price       | The starting price of the selling coin; it is proportional to the paying coin denom | 
-| min_bid_price     | The minimum bid price that bidders must provide                                     |
-| selling_coin      | The selling amount of coin for the auction                                          | 
-| paying_coin_denom | The paying coin denom that bidders use to bid with                                  | 
-| vesting_schedules | The vesting schedules that release the paying coins to the autioneer                | 
-| start_time        | The start time of the auction                                                       | 
-| end_time          | The end time of the auction                                                         | 
-|                   |                                                                                     |
+the auctioneer can create a fixed price auction to `1000,000,000,000denom1` coin, where the starting price is `2.0` which means that the price of `1denom1` is `2denom2`. The auction starts at `2022-02-01T00:00:00Z` and ends at `2022-03-01T00:00:00Z`.
```
Comment (jaybxyz) [inline docs/How-To/cli/README.md]: This description uses start price of 2.0 whereas the example uses 1.0. I think it is better to change one of them to follow the other.
```suggestion
An auctioneer creates a fixed price auction with `1_000,000,000,000denom1` selling coin where the start price is `2.0`. It means that the price of `1denom1` is `2denom2`. The auction starts at `2022-02-01T00:00:00Z` and ends at `2022-03-01T00:00:00Z`.
```

File: docs/How-To/cli/README.md [hunk 3]
```diff
@@ -11,35 +11,54 @@ This document provides a high-level overview of how the command line (CLI) inter
...
+| :---------------- | :---------------------------------------------------------------------------------- |
+| allowed_bidders | The list of allowed bidders that can participate in the auction, with a maximum possible bid amount for each bidder. It is empty when an auction is created. The module is designed to delegate permission to an external module to add its allowed bidders to the auction. |
+| start_price       | The starting price of the selling coin; it is proportional to the paying coin denom. This is the matched price. | 
+| selling_coin      | The selling amount of coin for the auction                                      | 
+| paying_coin_denom | The paying coin denom that bidders use to bid with                                  | 
+| vesting_schedules | The vesting schedules that release the paying coins to the autioneer                | 
+| start_time        | The start time of the auction                                                       | 
+| end_time          | The end time of the auction                                                         | 
+|                   |                                                                                     |
```
Comment (jaybxyz) [inline docs/How-To/cli/README.md]: I thought this line was needed to close the table, but it looks like it is adding another empty row. Can you delete this? https://github.com/jeonghoyeo7/fundraising/tree/128-fix-cli-placeBid/docs/How-To/cli#createfixedpriceauction
```suggestion
```

File: docs/How-To/cli/README.md [hunk 4]
```diff
@@ -86,79 +93,38 @@ fundraisingd tx fundraising create-fixed-price-auction auction.json \
...
+| max_extended_round   | The maximum number of extended rounds that provides additional opportunity for the bidders to place bids when more than a certain ratio of the number of the matched bids are reduced compared to the previous end time  |
+| extended_round_rate | The threshold reduction of the number of the matched bids are reduced compared to the previous end time to decide the necessity of another extended round | 
+
+Example of input as JSON:
 
 ```json
 {
-  "start_price": "0.500000000000000000",
+  "allowed_bidders": [],
+	"start_price": "2.000000000000000000",
```
Comment (jaybxyz) [inline docs/How-To/cli/README.md]: Needs to correct the indentation in this line.

File: docs/How-To/cli/README.md [hunk 5]
```diff
@@ -422,83 +245,65 @@ fundraisingd tx fundraising bid 1 1.0 5000000denom2 \
...
-}
+Usage
+```bash
+modify-bid [auction-id] [bid-id] [price] [coin]
+```
+
+| **Argument**      |  **Description**                     |
+| :---------------- | :----------------------------------- |
+| auction-id        | auction ID that the bid corresponds to. | 
+| bid-id    | bid ID that the bidder placed previously in this auction |
```
Comment (jaybxyz) [inline docs/How-To/cli/README.md]: I don't think we need to capitalize id imo (description).

File: x/fundraising/keeper/auction.go [hunk 6]
```diff
@@ -244,7 +244,7 @@ func (k Keeper) CreateBatchAuction(ctx sdk.Context, msg *types.MsgCreateBatchAuc
 
 	ctx.EventManager().EmitEvents(sdk.Events{
 		sdk.NewEvent(
-			types.EventTypeCreateFixedPriceAuction,
+			types.EventTypeCreateBatchAuction,
```
Comment (jaybxyz) [inline x/fundraising/keeper/auction.go:247]: Great catch!

File: x/fundraising/spec/02_state.md [hunk 7]
```diff
@@ -145,10 +145,10 @@ type FixedPriceAuction struct {
 type BatchAuction struct {
     *BaseAuction
     
-	MinBidPrice           sdk.Dec           // the minimum bid price that bidders must provide
-	MatchedPrice          sdk.Dec           // the matched price of the auction (a.k.a., winning price)
+	MinBidPrice			sdk.Dec	// the minimum bid price that bidders must provide
+	MatchedPrice		sdk.Dec	// the matched price of the auction (a.k.a., winning price)
```
Comment (jaybxyz) [inline x/fundraising/spec/02_state.md:149]: Can you check the indentation? 
https://github.com/jeonghoyeo7/fundraising/blob/128-fix-cli-placeBid/x/fundraising/spec/02_state.md

**Review by jaybxyz — CHANGES_REQUESTED** [review]
[Markdown link checker](https://github.com/tendermint/fundraising/runs/5627139708?check_suite_focus=true#step:4:60) is failing. Can you check?

File: x/fundraising/spec/02_state.md (hunk 7 above)
Comment (jeonghoyeo7) [reply x/fundraising/spec/02_state.md:149]: Checked and modified.

File: docs/How-To/cli/README.md (hunk 5 above)
Comment (jeonghoyeo7) [reply docs/How-To/cli/README.md]: Checked, and agree to modify with small letters.

File: docs/How-To/cli/README.md (hunk 3 above)
Comment (jeonghoyeo7) [reply docs/How-To/cli/README.md]: Removed.

File: docs/How-To/cli/README.md (hunk 2 above)
Comment (jeonghoyeo7) [reply docs/How-To/cli/README.md]: I checked and modified as suggested.

File: docs/How-To/cli/README.md (hunk 1 above)
Comment (jeonghoyeo7) [reply docs/How-To/cli/README.md]: I checked and modified.

File: docs/How-To/cli/README.md (hunk 4 above)
Comment (jeonghoyeo7) [reply docs/How-To/cli/README.md]: Checked and fixed this.

**Pull request merged**